src/linear_algebra/affine_space.lean

```diff
@@ -6,6 +6,8 @@ Author: Joseph Myers.
 import algebra.add_torsor
 import data.indicator_function
 import linear_algebra.basis
+import analysis.normed_space.finite_dimension
+import topology.algebra.continuous_functions
```

Comment:
There don't seem to be any existing (direct) imports of `analysis` or `topology` files from `algebra` or `linear_algebra`. Maybe the topological parts of the changes to this file actually belong somewhere else under `analysis` or `topology`?

Reply:
Yes, I was wondering where I should put those lemmas. They're about continuity of affine maps, and I don't think there's anything else in mathlib about this. So now I put them alongside other properties of affine maps, but they're the only ones that reference topology in the bunch. I guess the other option would be to put them somewhere under `analysis/normed_space`, where similar properties are found for linear maps. Maybe in a new file called `analysis/normed_space/affine.lean` or something like that...?

Reply:
For things depending on a norm, `analysis/normed_space/add_torsor.lean` is the closest we have, but `normed_add_torsor` is specifically for the case where a metric is given on the points that agrees with that induced by the underlying `normed_group`. For the part involving the topology but not a norm, maybe a new file in `topology/algebra` would make sense.

Reply:
I have now moved those two continuity lemmas to the new file `topology/algebra/affine.lean`, and I managed to get rid of the norm stuff entirely.